.github/workflows/weekly-release-pr.yml

```
@@ -0,0 +1,65 @@
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          DATE=$(TZ=Asia/Taipei date +%Y-%m-%d)
+          COMMITS_AHEAD=${{ steps.check.outputs.commits_ahead }}
+          gh pr create \
+            --base main \
+            --head develop \
+            --title "[release] develop → main ($DATE)" \
+            --body "## Weekly Release — $DATE
+
+Auto-generated by weekly-release-pr workflow.
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)